### ISSUE TITLE
remove "cloudfoundry/docs-bosh (stars) and (forks)" from top corner

### ISSUE THREAD

**Issue by cppforlife** [bosh-io/web#29]
looks misleading as if docs-bosh is the primary repo.

**Comment by cppforlife** [comment]
cc @dpb587-pivotal

**Comment by drnic** [comment]
Perhaps change it to show case cloudfoundry/bosh project forks/stars instead?

**Comment by drnic** [comment]
One of the things I love about the mkdocs-material theme is every page has a pencil icon which goes straight to an "Edit this file on github" URL. This has been huge for https://concoursetutorial.com and https://ultimateguidetobosh.com over the last 6 months.

**Comment by dpb587-pivotal** [comment]
Agreed, although technically the old site had a "Contribute changes to this page" link in the footer which did the same and people found useful as well.

To be clear... I changed the top corner to refer to cloudfoundry/bosh, but the pencil edit links for doc changes still work for docs-bosh/bpm-release repos like normal.
